Src/Clients/WebUI/Controllers/Storage/GenresWebUiController.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using TeamProject.Application.Common.Interfaces.Infrastructure;
+using TeamProject.Clients.Common;
+using TeamProject.Clients.Common.Models.Storage.Genres;
+using TeamProject.Clients.WebUI.Controllers.Common;
+
+namespace TeamProject.Clients.WebUI.Controllers.Storage
+{
+    public class GenresWebUiController : BaseController
+    {
+        private readonly IApiTools _apiTools;
+        private readonly IAuthorizeApiTools _authorizeApiTools;
+        public GenresWebUiController(IApiTools apiTools, IAuthorizeApiTools authorizeApiTools)
+        {
+            _apiTools = apiTools;
+            _authorizeApiTools = authorizeApiTools;
+        }
+        [HttpGet]
+        public IActionResult Index()
+        {
+            return View();
+        }
+        public async Task<IActionResult> Genres()
+        {
+            var model = await _authorizeApiTools.FetchAsync<List<GenreBindingModel>>(
+                CommonClientsDefaults.WebApiGenresControllerGetAll,JwtToken);
+            return PartialView(model);            
+        }
+        [HttpGet]
+        public ActionResult CreateGenre()
+        {
+            return View();
+        }
+        [HttpPost]
+        public async Task<IActionResult> CreateGenre(GenreBindingModel genre)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add the 'Async' suffix to the name of this method.](https://app.codacy.com/manual/liannoi/team-project/pullRequest?prid=5384135)